plutus-core/plutus-ir/src/PlutusIR/Compiler/Definitions.hs

```diff
@@ -311,3 +350,27 @@ lookupDestructor x name = do
   pure $ case Map.lookup name ds of
     Just (PLC.Def{PLC.defVal = (Datatype _ _ _ destr _)}, _) -> Just $ Var x destr
     Nothing                                                  -> Nothing
+
+lookupManualDestructor ::
```

Comment:
Since this looks up both manual destructors and regular destructors, you can combine this with `lookupDestructor`, and keep the `lookupDestructor` name.